workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MigrationAssessmentWorkflow.java

```diff
@@ -0,0 +1,97 @@
+import com.redhat.parodos.tasks.migrationtoolkit.GetAnalysisTask;
+import com.redhat.parodos.tasks.migrationtoolkit.GetApplicationTask;
+import com.redhat.parodos.tasks.migrationtoolkit.SubmitAnalysisTask;
+import com.redhat.parodos.workflow.annotation.Assessment;
+import com.redhat.parodos.workflow.annotation.Checker;
+import com.redhat.parodos.workflow.option.WorkFlowOption;
+import com.redhat.parodos.workflow.task.infrastructure.Notifier;
+import com.redhat.parodos.workflows.workflow.WorkFlow;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import static com.redhat.parodos.workflows.workflow.SequentialFlow.Builder.aNewSequentialFlow;
+
+/**
+ * An assessment workflow to analyze applications using Migration Toolkit for Applications
+ * and return a move2kube option when needed.
+ * <p>
+ * This workflow will: - create an application with a name, and git repo URL - submit an
+ * analysis report for it - wait till the report is back, analyze it, and return a
+ * workflow option to move2kube based on the findings
+ * <p>
+ */
+@Configuration
+@Profile("dev")
+public class MigrationAssessmentWorkflow {
+
+	// the url of the MTA (migration toolkit for application)
+	private String mtaUrl = Objects.requireNonNull(System.getenv("MTA_URL"));
+
+	@Bean
+	public CreateApplicationTask createApplicationTask() {
+		return new CreateApplicationTask(URI.create(mtaUrl), "");
+	}
+
+	@Bean
+	public GetApplicationTask getApplicationTask() {
+		return new GetApplicationTask(URI.create(mtaUrl), "");
+	}
+
+	@Bean
+	public SubmitAnalysisTask submitAnalysisTask(WorkFlow fetchReportURL) {
+		SubmitAnalysisTask t = new SubmitAnalysisTask(URI.create(mtaUrl), "");
+		t.setWorkFlowCheckers(List.of(fetchReportURL));
+		return t;
+	}
+
+	@Bean
+	public GetAnalysisTask getAnalysisTask(Notifier notifier) {
+		return new GetAnalysisTask(URI.create(mtaUrl), "", notifier);
+	}
+
+	@Bean
+	ProcessAnalysisTask processAnalysisTask(WorkFlowOption move2kube, WorkFlowOption defaultOption) {
+		return new ProcessAnalysisTask(move2kube, defaultOption,
+				analysisIncidents -> analysisIncidents.mandatory() > 0);
+	}
+
+	@Bean
+	WorkFlowOption move2kube() {
+		return new WorkFlowOption.Builder("move2kube", "move2kube").addToDetails("Migration to OCP")
+				.displayName("Migration to OCP")
+				.setDescription("This app is ready to be migrated to OCP. Click to migrate.").build();
+	}
+
+	@Bean
+	WorkFlowOption defaultOption() {
+		return new WorkFlowOption.Builder("defaultOption", "AnalyzeApplicationAssessment")
+				.addToDetails("Rerun Analysis").displayName("Run Migration Analysis")
+				.setDescription(
+						"This application didn't meet the expected analysis score. Update or apply the relevant fixes and re-run the analysis")
+				.build();
+	}
+
+	@Bean(name = "AnalyzeApplicationAssessment")
+	@Assessment
```

Comment:
should repoUrl and appName parameters be here?

Reply:
yeah I set them as workflow level params